src/user-event/press/utils/warnAboutRealTimers.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { warnAboutRealTimers } from './warnAboutRealTimers';
+
+const mockConsoleWarn = jest.spyOn(console, 'warn').mockImplementation();
+
+test('logs a warning about usign real timers with user event', () => {
+  warnAboutRealTimers();
+
+  expect(mockConsoleWarn.mock.calls[0][0]).toMatchInlineSnapshot(`
+    "It is not recommended to use userEvent without using fake timers
```

Comment:
[nit] Perhaps it's worth removing double negation: "It is recommended to use `userEvent` with fake timers."